pkg/berglas/resolver.go

```diff
@@ -86,11 +86,10 @@ func Replace(ctx context.Context, key string) error {
 	return client.Replace(ctx, key)
 }
 
-// Replace parses a berglas reference from the environment variable at the
-// given environment variable name. If parsing and extraction is successful,
-// this function replaces the value of the environment variable to the resolved
-// secret reference.
-func (c *Client) Replace(ctx context.Context, key string) error {
+// ReplaceValue parses a berglas reference from value. If parsing and extraction
+// is successful, this function sets the value of the environment variable to the
+// resolved secret reference.
+func (c *Client) ReplaceValue(ctx context.Context, key string, value string) error {
 	plaintext, err := c.Resolve(ctx, os.Getenv(key))
```

Comment:
This PR neglected to actually use `value` here. This call to `os.Getenv(key)` should have been `value`.

Reply:
Ah damnit you're right.

Reply:
Opened new PR #53